docs/jade-getting-started.md

```diff
@@ -0,0 +1,219 @@
+line interface:
+
+```
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+Once Homebrew is installed, there are a number of useful development tools that
+should be installed.
+
+1. [Git](https://git-scm.com/) is a version control tool for tracking changes in
+projects and code:
+
+```
+brew install git
+```
+
+2. [jq](https://stedolan.github.io/jq/) is a command line JSON processing tool:
+
+```
+brew install jq
+```
+
+3. [Docker](https://www.docker.com/) is a tool to deliver software in packages
+called containers. Docker for MacOS also includes [Kubernetes](https://kubernetes.io/),
+which deploys groups of containers together in clusters. Once installed, you'll
+need to run Docker once from your list of Applications:
+
+```
+brew cask install docker
+open -a Docker
+```
+
+4. [Helm](https://helm.sh/) streamlines the process of defining, installing, and
+upgrading Kubernetes deployments, which are otherwise challenging to manage:
+
+```
+brew install helm
+helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+helm repo add datarepo-helm https://broadinstitute.github.io/datarepo-helm
+helm plugin install https://github.com/thomastaylor312/helm-namespace
+helm repo update
+```
+
+5. [Skaffold](https://github.com/GoogleContainerTools/skaffold) facilitates the
+continuous development of Kubernetes resources:
+
+```
+brew install skaffold
+```
+
+6. [Vault](https://www.vaultproject.io/) is an encrypted database used to store
+many of the team's secrets such as keys and passwords:
+
+```
+brew install vault
+export VAULT_ADDR=https://clotho.broadinstitute.org:8200
+```
+
+7. Much of the code written at the Broad is in [Java](https://en.wikipedia.org/wiki/Java_(programming_language)).
+Install the OpenJDK 8 runtime from the [AdoptOpenJDK](https://adoptopenjdk.net/)
+project to develop and run Java code:
+
+```
+brew tap AdoptOpenJDK/openjdk
+brew cask install adoptopenjdk8
+```
+
+8. [Google Cloud SDK](https://cloud.google.com/sdk) is a command-line interface
+to Google Cloud services. Once it is installed, you'll need to allow auth access
+and configure Docker to connect to the appropriate Google Cloud endpoint when
+necessary:
+
+```
+brew cask install google-cloud-sdk
+gcloud auth login
+gcloud auth application-default login
+gcloud auth configure-docker
+```
+
+9. [IntelliJ IDEA](https://www.jetbrains.com/idea/) is an integrated development
+environment (IDE) for Java. There are two versions available: **Ultimate** (paid)
+and **Community** (open-source). The **Community** edition has all the features
+needed for development:
+
+```
+brew cask install intellij-idea-ce
+open -a "IntelliJ IDEA CE"
+```
+
+10. Once it is launched, go to IntelliJ IDEA -> Preferences -> Plugins,
+then click in the search box and install **Cloud Code**, which integrates
+Google Cloud features with IntelliJ IDEA.
+
+## 4. Install Postgres 9.6
```

Comment:
YMMV, but I have Postgres 11 installed and haven't run into any issues

Reply:
We might be able to switch it to Postgres 11 now, I think we were having some issues with some of the tests earlier.

Reply:
Switched to Postgres 12, since it's the latest stable release.